runners/spark/src/test/java/org/apache/beam/runners/spark/PipelineRule.java

```diff
@@ -33,34 +33,29 @@
  */
 public class PipelineRule implements TestRule {
 
-  private final TestName testName = new TestName();
-
   private final SparkPipelineRule delegate;
   private final RuleChain chain;
 
-  private PipelineRule() {
-    this.delegate = new SparkPipelineRule(testName);
-    this.chain = RuleChain.outerRule(testName).around(this.delegate);
-  }
-
-  private PipelineRule(Duration forcedTimeout) {
-    this.delegate = new SparkStreamingPipelineRule(forcedTimeout, testName);
+  private PipelineRule(SparkPipelineRule delegate) {
```

Comment:
Well, that's me being "over protective" 😄 
`PipelineRule` might not work for all test scenarios, so just in case. Better to remove an empty directory (or none) rather then risking junk in Jenkins.
Would you agree ?